web-components/src/mixins/FocusTrapMixin.ts

```diff
@@ -279,6 +280,21 @@ export const FocusTrapMixin = <T extends AnyConstructor<FocusClass & FocusTrapCl
       }
     }
 
+    protected setFocusOnTrigger(triggerElement: HTMLElement) {
```

Comment:
Please leave a comment to describe what this function actually achieve.

Reply:
done